### PR TITLE
fix: exploration exp

### PR DESCRIPTION
**Pull request by AngeloTadeucci**

### CODE REVIEW

File: Maple2.Server.Game/Manager/ExperienceManager.cs [hunk 1]
```diff
@@ -112,7 +112,10 @@ public sealed class ExperienceManager {
             return;
         }
 
-        long expValue = 0;
+        long expValue;
+        short charLevel = session.Player.Value.Character.Level;
+        int fieldLevel = session.Field.Metadata.Drop.Level;
+        int correctedLevel = fieldLevel > charLevel ? charLevel : fieldLevel;
```
Comment (Zintixx) [inline Maple2.Server.Game/Manager/ExperienceManager.cs:118]: IF these are only used in a specific case in the switch, let's move it within there.
Also, `charLevel` is redundant. you can already get that from `Level` in the ExperienceManager.

File: Maple2.Server.Game/Manager/ExperienceManager.cs [hunk 2]
```diff
@@ -143,6 +147,19 @@ public sealed class ExperienceManager {
...
+            return;
+        }
+
+        if (!expBase.TryGetValue(moblevel, out long expValue)) {
+            return;
+        }
+
+
+        AddExp((long) (expValue * modifier) + additionalExp, ExpType.monster.Message());
+    }
```
Comment (Zintixx) [inline Maple2.Server.Game/Manager/ExperienceManager.cs:161]: I wonder if we just combined this with the method above? I believe there is a `monster` ExpType value

And change the parameters to include `level`?

**Review by Zintixx — CHANGES_REQUESTED** [review]

File: Maple2.Server.Game/Manager/ExperienceManager.cs (hunk 2 above)
Comment (AngeloTadeucci) [reply Maple2.Server.Game/Manager/ExperienceManager.cs:161]: but we dont have factor for monster

Comment (Zintixx) [reply Maple2.Server.Game/Manager/ExperienceManager.cs:161]: ah yeah. nevermind

File: Maple2.Server.Game/Manager/ExperienceManager.cs (hunk 1 above)
Comment (AngeloTadeucci) [reply Maple2.Server.Game/Manager/ExperienceManager.cs]: fixed

**Review by Zintixx — APPROVED** [review]

**Pull request merged**